src/views/Help/ITGovernance/index.tsx

```diff
@@ -1,18 +1,48 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import { Route, Switch } from 'react-router-dom';
+import { CardGroup } from '@trussworks/react-uswds';
 
+import ArticleCard from 'components/ArticleCard';
+import HelpBreadcrumb from 'components/HelpBreadcrumb';
+import HelpContacts from 'components/HelpContacts';
+import HelpPageIntro from 'components/HelpPageIntro';
+import Divider from 'components/shared/Divider';
+
+import itGovernanceArticles from './articles';
 import PrepareForGRT from './PrepareForGRT';
 
+const PageContent = () => {
+  const { t } = useTranslation('help');
+  return (
+    <>
+      <HelpBreadcrumb type="Back" />
+      <HelpPageIntro
+        heading={t('itGovernance.heading')}
+        subheading={t('itGovernance.subheading')}
+      />
+      <CardGroup className="padding-top-1 padding-bottom-4">
+        {itGovernanceArticles.map(article => {
+          return <ArticleCard key={article.route} {...article} tag={false} />;
+        })}
+      </CardGroup>
+      <Divider />
+      <HelpContacts type="IT Governance" />
+    </>
+  );
+};
+
 const ITGovernance = () => {
   return (
-    <div className="grid-container">
+    <>
       <Switch>
         <Route
           path="/help/it-governance/prepare-for-grt"
           render={() => <PrepareForGRT />}
         />
+        <Route path="/help/it-governance" render={() => <PageContent />} />
```

Comment:
The route is already defined in the home `index.tsx`.  You should be able to just insert the `<PageContent />` component directly in the jsx, without the need to define and render the component